roles/chrony_exporter/molecule/alternative/tests/test_alternative.py

```diff
@@ -37,10 +37,9 @@ def test_protecthome_property(host):
     assert p.get("ProtectHome") == "yes"
 
 
-def test_socket(host):
-    sockets = [
-        "tcp://127.0.0.1:8080"
-    ]
-    for socket in sockets:
-        s = host.socket(socket)
-        assert s.is_listening
+@pytest.mark.parametrize("sockets", [
```

Comment:
You added a pytest, but forgot to `import pytest` at the top.

Reply:
thanks, I assumed that was already imported by all the test files :facepalm: